README.md

```diff
@@ -245,7 +245,7 @@ Additional lists you might find useful:
 - [CakeAdmin plugin](https://github.com/cakemanager/cakephp-cakeadmin) - A non-stable user management plugin with a built-in admin area.
 - [CakeDC/Enum plugin](https://github.com/CakeDC/enum) - A plugin to add enumeration list support to your app.
 - :strawberry: [CakeDto plugin](https://github.com/dereuromark/cakephp-dto) - Quickly generate useful data transfer objects for your app (mutable/immutable), replacing messy arrays and leveraging your IDE through typehinting and autocomplete.
-- [CakeImpersonate plugin](https://github.com/jomweb/CakeImpersonate) - A component that stores the current authentication session and creates new session for impersonating Users. User can revert back to original authentication sessions without the need to re-login.
+- :strawberry: [CakeImpersonate plugin](https://github.com/jomweb/CakeImpersonate) - A component that stores the current authentication session and creates new session for impersonating Users. User can revert back to original authentication sessions without the need to re-login.
```

Comment:
I don't see how this plugin's `4.0.0-beta` release is Cake 4 compatible when it's composer.json has this https://github.com/jomweb/CakeImpersonate/blob/4.0.0-beta/composer.json#L8

Reply:
Ahh good catch! Must have tagged the wrong branch. But https://github.com/jomweb/CakeImpersonate/releases/tag/4.0.0-beta1 is now available with the correct tagged branch.